Index.html

```diff
@@ -0,0 +1,322 @@
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="stylesheet" href="style.css">
+        <title>Figma-portfolio</title>
+    </head>
+    <body>
+      <header class="header">
+        <a href="#" id="logo">My Logo</a>
+        <nav>
+          <ul id="nav">
+            <li class="nav-link"><a href="#">Portfolio</a></li>
+            <li class="nav-link"><a href="#">About</a></li>
+            <li class="nav-link"><a href="#">Contact</a></li>
+          </ul>
+          <div class="mobile-nav">
+            <div id="menu">
+              <i class="bar"></i>
+              <i class="bar"></i>
+              <i class="bar"></i>
+            </div>
+            <ul id="link">
+              <li class="mobile-link">
+                <a href="#">Portfolio</a>
+              </li>
+              <li class="mobile-link">
+                <a href="#">About</a>
+              </li>
+              <li class="mobile-link">
+                <a href="#">Contact</a>
+              </li>
+            </ul>
+          </div>
+        </nav>
+      </header>
+      <section class="headline">
+        <div id="head-container">
+          <h1 class="head">
+            I'm John.   </br>
+            Glad to see </br>
+            you!
+          </h1>
+          <p class="resume">
+            I’m a software developer! I can help you </br>
+            build a product, feature orwebsite Look </br>
+            through some of my work and </br>
+            experience. If you like what you see and </br>
+            have a project you need coded, don’t </br>
+            hesitate to contact me! 
+          </p>
+          <h2 class="connect">LET’S CONNECT</h2>
+          <ul class="social-media-icons">
+            <li>
+              <a rel="noopener" href="https://twitter.com/jovasal81" target="_blank">
+                <img src="./images/twitter.png" alt="twitter" />
+              </a>
+            </li>
+            <li>
+              <a rel="noopener" href="https://www.linkedin.com/in/john-alonso-vasquez-salcedo-95749632/" target="_blank">
+                <img src="images/Linkedin.png" alt="linkedIn" />
+              </a>
+            </li>
+            <li>
+              <a rel="noopener" href="https://medium.com/@jovasal" target="_blank">
+                <img src="images/medium.png" alt="medium" />
+              </a>
+            </li>
+            <li>
+              <a rel="noopener" href="https://github.com/jovasal" target="_blank">
+                <img src="images/github.png" alt="git" />
+              </a>
+            </li>
+            <li>
+              <a rel="noopener" href="https://wellfound.com/john-vasquez-12" target="_blank">
+                <img src="images/angel.png" alt="angel" />
+              </a>
+            </li>
+          </ul>
+        </div>
+      </section>
+      <main class="secpa">
```

Comment:
- [ ] kindly consider displaying all the cards in this section in the center so that the grey background is visible on the left side too to meet the design in the Figma check the images below for more clarification 👇 
- your design 👇 
<img width="325" alt="Screen Shot 2023-06-03 at 07 16 57" src="https://github.com/jovasal/Finish-Mob-Figma/assets/60146030/4f546b5c-bb59-4a24-9ba7-2a49b7eaaaa8">


- Figma design 👇 
<img width="354" alt="Screen Shot 2023-06-03 at 07 15 35" src="https://github.com/jovasal/Finish-Mob-Figma/assets/60146030/700b275d-7fb6-45df-b3e3-9a361f65be44">

Reply:
Thank you so much!!

HTML CSS JAVASCRIPT is ok now, Also All project are in center:

![image](https://github.com/jovasal/Finish-Mob-Figma/assets/35693330/e67e4065-f677-4408-8014-ec7e4f9031fe)
![image](https://github.com/jovasal/Finish-Mob-Figma/assets/35693330/c24bc7ad-2fc0-48cb-b98b-055ddf744ef2)
![image](https://github.com/jovasal/Finish-Mob-Figma/assets/35693330/c7e1ac52-2d64-415d-a523-54bc936d032a)
![image](https://github.com/jovasal/Finish-Mob-Figma/assets/35693330/8d0aab92-c301-4bff-af27-71cd56cf24cc)

All comments were fixed successfully.